### ISSUE TITLE
improve diff editor SR accessibility

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#203576]
@jooyoungseo mentioned that the diff editor experience is a little confusing because we don't explicitly state that the current editor is the modified one vs the original editor. 

AFAICT, there's also no way to jump between the original and modified ones. Is this correct?

@hediet & @isidorn, I'd appreciate your thoughts here. 

Edit: action items:

- [ ] add info in help dialog about switch sides command
- [ ] get accessibility signals to work in original editor

**Comment by isidorn** [comment]
There is the Diff Editor: Switch Sides command that moves focus to the other side. Maybe the wording could be improved.
So not sure what needs to be done here? Feels like similar to the other issue you created about missing clarity in announcements on original vs modified.

**Comment by meganrogge** [comment]
Yes, I was unaware of that command. We should include it in the diff editor help dialog and include a default keybinding for when in screen reader mode as that's definitely something users would want to do IMO.

**Comment by meganrogge** [comment]
@isidorn pointed out this should maybe be a tab stop if possible

**Comment by meganrogge** [comment]
could you tell there was a focus change at all? 

Is there also no difference if you run the command vs use the keybinding?

**Comment by meganrogge** [comment]
Audio cues do not work when I'm navigating in the original editor. 

![Image](https://github.com/microsoft/vscode/assets/29464607/8fff12af-d806-43ea-8d23-219240e9df92)
